base/io.jl

```diff
@@ -176,6 +176,19 @@ julia> write(io, "Sometimes those members") + write(io, " write documentation.")
 julia> String(take!(io))
 "Sometimes those members write documentation."
 ```
+Note that in general the value needs to be wrapped in a `Ref`, but for several basic types this can be skipped as above.
```

Comment:
I think this makes it sound like `Ref` is needed more often than it really is. In practice, most/all types with a sensible binary representation have direct `write` methods. In fact I'm not even sure this use of `Ref` is intended; it's buggy and will sometimes write references as their address.

Maybe I'd say "User-defined plain-data types without `write` methods can be written when wrapped in a Ref".

Reply:
Okay I changed it. I think there should be a default way to write bitstypes in `Base`. Whether it is via `write(io, Ref(obj))` or another function I don't have a strong opinion about.